### ISSUE TITLE
LayoutLM: Pre-training of Text and Layout for Document Image Understanding

### ISSUE THREAD

**Issue by RandyGen** [RandyGen/reading_papers#16]
# LayoutLM: Pre-training of Text and Layout for Document Image Understanding
https://arxiv.org/abs/1912.13318

## Abst
- スキャンした文書画像間のテキストとレイアウト情報の相互作用共同でモデル化するモデルの提案
    - テキストとレイアウトを一つのフレームワークで共同学習
- フォーム理解，領収書理解，文書画像分類で検証
    - sota

## Intro
- Document Intelligence
    - 情報ファイルをデジタル化したもの
    - 紙媒体をスキャンしたもの
        - ex. 発注書，報告書，メール，契約書，手紙，請求書など
- ビジネス文書理解の困難さ
    - レイアウト，フォーマットの多様性
    - 文書画像の品質の低さ
    - テンプレート構造の複雑さ
- 多くの企業が手作業でデータを抽出
    - この問題に対処するために文書AIの設計が進んでいる
    - 一方で2つの限界に直面
        - 大規模なラベルなし学習サンプルの必要性
        - テキストとレイアウト情報の事前学習の重要性
- LayoutLM
    - テキスト埋め込み
        - トークンの相対位置を示す2次元埋め込み
    - 　マルチタスク学習
        - 　Masked Visual-Language Model loss
        - 　Multi-label Document Classification loss
    - 　IDT-CDIP Test Collection 1.0で事前学習
        - 　600万以上のスキャン文書
        - 　1100万以上のスキャン文書画像
        - 　ex. 手紙，メモ，メール請求書，記事，プレゼン
    - 　下流タスク
        - 　FUNSD
        - 　SROIE(Scanned Receipts Information Extractoin)
        - 　RVL-CDIP

## Model
- 文書レイアウト情報
    - Key-Valueなどの情報
    - 2次元の位置情報を埋め込む
- 視覚情報
    - 従来のテキスト表現に画像特徴を組み合わせる
- アーキテクチャ
    - テキスト情報と2次元情報と画像埋め込みの組み合わせ
        - OCRで2次元情報
        - Faster R-CNNで画像領域特徴量
- Pretrain
    - テキスト情報と2次元情報を用いて表現を学習
    - 画像埋め込みからマルチラベル文書分類
- タスクに応じて トークン or CLS の分類問題になる

## Exp
- dataset
    - 事前学習
        - IIT-CDIP Test Collection 1.0
            - TesseractでOCR→スケーリング
    - fine-turning
        - FUNSD（フォーム理解）
            - ノイズの多いスキャン画像のKey-Value抽出
        - SROIE（レシート理解）
            - あらかじめ定義された情報の抽出
        - RVL-CDIP（文書画像分類）
            - グレースケールのカテゴリにおける文書分類

## Result
- FUNSD, SROIE, RVL-CDIPで精度向上

## Related Work
- ルールベース
    - コスト
- 機械学習
    - 統計量からの作成
- DL

## Conclusion
- トークン，レイアウト，画像等の埋め込みを入力するマルチモーダル
- 幾つかの下流タスクで有効的
